docs/chapters/03.txt

```diff
@@ -745,13 +751,23 @@ value is, but also on whether your DBMS uses rounding or truncation to turn it
 into an integer. Here are the choices (note that "rounding toward zero" is
 really truncating):
 
-::
++----------+-------------+-------------+------------+------------+
+| ::       | ::          | ::          | ::         | ::         |
```

Comment:
all of the cells in this row should be marked up as normal text, so you can drop the :: line and re-indent them so that they are one " " space away from the left hand | line